model/Image.php

```diff
@@ -564,6 +564,11 @@ public function generateSetSize(Image_Backend $backend, $width, $height) {
 		return $backend->paddedResize($width, $height);
 	}
 
+	/**
+	 * TODO: Documentation.
```

Comment:
I'm sorry -- I didn't have the time to research it but I did want to point it out. I'll paste that in; otherwise a bit too busy, I apologize :(